docs/plugins/random-sampling.md

```diff
@@ -1,16 +1,16 @@
 The Random Sampling plugin allows random sampling for cardinality-based feature models.
-It generates a custom amount of random valid configurations and outputs them into the console.
+It generates a custom number of random valid configurations and outputs them into the console.
 
 The Random Sampling plugin requires the model to be bound which means no infinite upper bounds as instance cardinalities are allowed.
 In case of an unbound model, you can use other plugins like the Big M plugin to replace infinte upper bounds with finite ones.
 
 ## Usage
 
 Import a cfm and generate 5 random samples for it:
-The `--amount` parameter defaults to `1` if not specified.
+The `--number` parameter defaults to `1` if not specified.
```

Comment:
this should be changed as well (as well as down below)

Reply:
oops :O